src/api/entities/Instruction/types.ts

```diff
@@ -6,8 +6,7 @@ import { EventIdentifier } from '~/types';
 export enum InstructionStatus {
   Pending = 'Pending',
   Failed = 'Failed',
-  Rejected = 'Rejected',
-  Success = 'Success',
+  Executed = 'Executed',
```

Comment:
This is a good example on what awful dual version supporting is. 
When we decide to remove v5 support, we'll need to reintroduce this breaking change.